src/modules/layout/components/Header.js

```diff
@@ -112,6 +111,21 @@ class Header extends React.PureComponent<PropsType> {
     })
   }
 
+  showLanguageChange (): boolean {
+    const {routeKey, routeMapping} = this.props
+    const route = routeMapping[routeKey]
+
+    if (!route) {
+      return false
+    }
+
+    return !route.peek
```

Comment:
I'm still not quite sure why we should disable this on peek

Reply:
Because we do not want to allow changing the language when peeking a screen. Even though it would be technically possible it is not defined what should happen.

It is also not sure which instance to search in.

Reply:
Behavior could be changed in a separate task if we decide to allow this functionality.